### PR TITLE
Always display invalid-feedback elements

### PR DESCRIPTION
**Pull request by StijnVrolijk**

### CODE REVIEW

File: src/sass/components/_forms.scss [hunk 1]
```diff
@@ -150,3 +150,22 @@ label abbr[title] {
 .has-error .help-block {
   color: theme-color('danger');
 }
+
+.alert .error-list-item {
+  @extend .my-1;
```
Comment (mathiashelin) [inline src/sass/components/_forms.scss]: Use `margin-top: map-get($spacers, 1);` and `margin-bottom: map-get($spacers, 1);` instead. Extending should be avoided if it's not necessary.

File: src/sass/components/_forms.scss [hunk 2]
```diff
@@ -150,3 +150,22 @@ label abbr[title] {
...
+}
+
+// Display an invalid select2 as invalid
+select.is-invalid + .select2 .select2-selection {
+  @extend .form-control.is-invalid
+}
+
+// Make sure input-group-texts in an append have a radius, even if an (in)valid-feedback element comes after it
+.input-group-append .input-group-text {
+  border-top-right-radius: 0.25rem !important;
```
Comment (mathiashelin) [inline src/sass/components/_forms.scss]: Use `$border-radius`.

File: src/sass/components/_forms.scss [hunk 3]
```diff
@@ -150,3 +150,22 @@ label abbr[title] {
...
+  border-bottom-right-radius: 0.25rem !important;
```
Comment (mathiashelin) [inline src/sass/components/_forms.scss]: Use `$border-radius`.

Comment (StijnVrolijk) [reply src/sass/components/_forms.scss]: 722b1b5

File: src/sass/components/_forms.scss (hunk 2 above)
Comment (StijnVrolijk) [reply src/sass/components/_forms.scss]: 722b1b5

File: src/sass/components/_forms.scss (hunk 1 above)
Comment (StijnVrolijk) [reply src/sass/components/_forms.scss]: 5da2802

**Pull request merged**